lib/utils/matrix_sdk_extensions/download_file_extension.dart

```diff
@@ -0,0 +1,169 @@
+import 'dart:io';
+
+import 'package:dio/dio.dart';
+import 'package:fluffychat/data/network/media/media_api.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/utils/matrix_sdk_extensions/event_extension.dart';
+import 'package:fluffychat/utils/storage_directory_utils.dart';
+import 'package:matrix/matrix.dart';
+
+extension DownloadFileExtension on Event {
+  bool canContainAttachment() {
+    return [EventTypes.Message, EventTypes.Sticker].contains(type);
+  }
+
+  bool isSending() {
+    return status.isSending;
+  }
+
+  Uri? getAttachmentOrThumbnailMxcUrl({bool getThumbnail = false}) {
+    return attachmentOrThumbnailMxcUrl(getThumbnail: getThumbnail);
+  }
+
+  bool isEncryptionDisabled(bool isEncrypted) {
+    return isEncrypted && !room.client.encryptionEnabled;
```

Comment:
I think `isEncrypted` name has not much sense it should be the reverse `isNotEncrypted` or `!isEncrypted` depending on what argument you pass

Reply:
the argument `isEncrypted` means is the file is encrypted or not, so it has correct meaning :)